virttest/utils_net.py

```diff
@@ -3376,6 +3385,21 @@ def get_host_default_gateway():
     return output
 
 
+def get_host_default_iface():
+    """
+    Get the Default interface in host.
+    :return: a string of the host's default interface.
+    :rtype: string
+    """
+    cmd = "ip route | awk '/default/ { print $5 }'"
+    try:
+        output = process.system_output(cmd, shell=True)
+    except:
+        raise exceptions.TestError("Failed to get the host's default interface.")
```

Comment:
How about merge this function to ```get_host_default_gateway()```, like:
```
def get_host_default_gateway(iface_name=False):

    if iface_name:
        ...
    ...
```

And instead of raising TestError, maybe we can log an error message(including exception message) here and return None, so when calling this function, we don't need to catch the exception again.